src/frontend/src/components/common/Filter/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useState } from 'react';
+import {
+  MinistryList,
+  PiaDrafterFilterList,
+  PiaStatuses,
+  PiaStatusList,
+} from '../../../constant/constant';
+import { isMPORole } from '../../../utils/helper.util';
+import Dropdown from '../Dropdown';
+import { IFilter } from './interfaces';
+
+const Filter = ({ id }: IFilter) => {
+  const [ministry, setMinistry] = useState<string>('');
+  const [status, setStatus] = useState<string>('');
+  const [drafterFilter, setDrafterFilter] = useState<string>('');
+
+  const handlePiaStatusChange = (newStatus: any) => {
+    setStatus(newStatus.target.value);
```

Comment:
From the looks of the implementation, is it pending API integration?

Reply:
yes